<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>学生信息</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="header">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/logo/header.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="display_group">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,6">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string/>
                </property>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="show_data">
                   <property name="text">
                    <string>显示数据</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="clear_data">
                   <property name="text">
                    <string>清空数据</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string>设置时间</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="text">
                    <string>PushButton</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="info_group">
                <property name="title">
                 <string>统计信息</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="stus_info"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>日志</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>操作日志</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableView" name="operation_list"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="title">
           <string>消费日志</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableView" name="consume_list"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="search_menu">
    <property name="title">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QMenu" name="stu_menu">
    <property name="title">
     <string>学生操作</string>
    </property>
   </widget>
   <widget class="QMenu" name="card_menu">
    <property name="title">
     <string>卡操作</string>
    </property>
   </widget>
   <widget class="QMenu" name="all_menu">
    <property name="title">
     <string>批量操作</string>
    </property>
   </widget>
   <widget class="QMenu" name="log_menu">
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QMenu" name="canteen_menu">
    <property name="title">
     <string>食堂</string>
    </property>
   </widget>
   <addaction name="search_menu"/>
   <addaction name="stu_menu"/>
   <addaction name="card_menu"/>
   <addaction name="canteen_menu"/>
   <addaction name="log_menu"/>
   <addaction name="all_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="search_person">
   <property name="text">
    <string>查找学生</string>
   </property>
  </action>
  <action name="Input_all">
   <property name="text">
    <string>批量导入</string>
   </property>
  </action>
  <action name="issue_all_card">
   <property name="text">
    <string>批量发卡</string>
   </property>
   <property name="toolTip">
    <string>为符合条件的账户发卡</string>
   </property>
  </action>
  <action name="open_account">
   <property name="text">
    <string>开户</string>
   </property>
  </action>
  <action name="remove_person">
   <property name="text">
    <string>销户</string>
   </property>
  </action>
  <action name="report_lost">
   <property name="text">
    <string>挂失</string>
   </property>
  </action>
  <action name="remove_lost">
   <property name="text">
    <string>解挂</string>
   </property>
  </action>
  <action name="issue_card">
   <property name="text">
    <string>发卡</string>
   </property>
  </action>
  <action name="add_money">
   <property name="text">
    <string>充值</string>
   </property>
  </action>
  <action name="open_log">
   <property name="text">
    <string>载入日志</string>
   </property>
  </action>
  <action name="consume">
   <property name="text">
    <string>消费</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
